js/main.js

```diff
@@ -1,10 +1,11 @@
 $("#more").click(function(){
     $("section").css({"position": "fixed"});
-    $("#info").animate({"left": "10%",}, 200);
-    $("#info").animate({"left": "0%",}, 200);
+    $("#info").animate({"left": "0%"}, 200);
```

Comment:
@ankitr that was the bounce effect. @gmittal suggested that I remove it to keep the animation serious. This isn't up to date?!